### PR TITLE
chore: added title to Veolas for duplicate title error in SEMrush

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

Just added a title to VeOlas to resolve SEMrush issues, duplicate title tags:
image

### CODE REVIEW

File: apps/govern/components/VeOlas/index.tsx [hunk 1]
```diff
@@ -23,53 +25,64 @@ const Title = styled.h1`
...
-                setIsIncreaseModalVisible(true);
-              } else {
-                setIsCreateLockModalVisible(true);
-              }
-            }}
-          >
-            Lock OLAS for veOLAS
-          </Button>
+    <>
+      <Head>
```
Comment (Tanya-atatakai) [inline apps/govern/components/VeOlas/index.tsx:37]: could you please add the whole meta block to govern app similar to one that we have on olas website or contribute or in bond app here apps/bond/pages/_app.jsx? currently govern app is not searchable in google :( I probably forgot to add that when created the app

**Review by Tanya-atatakai — CHANGES_REQUESTED** [review]
1) need to fix linter, github actions are failed
2) don't understand what it means "duplicated titles" in SEMrush, currently there's only one on veOLAS page 🤔 
3) we didn't fix it for the contracts page, is that expected?

File: apps/govern/components/VeOlas/index.tsx [hunk 2]
```diff
@@ -23,53 +25,64 @@ const Title = styled.h1`
...
+        <title>VeOlas</title>
```
Comment (Tanya-atatakai) [inline apps/govern/components/VeOlas/index.tsx:38]: should be `veOLAS`

**Review by Tanya-atatakai — CHANGES_REQUESTED** [review]
1) need to fix linter, github actions are failed
2)

File: apps/govern/pages/veolas.tsx [hunk 3]
```diff
@@ -1,3 +1,15 @@
-import { VeOlasPage } from '../components/VeOlas';
+import Meta from 'components/Meta';
+import { VeOlas } from '../components/VeOlas';
+
+const VeOlasPage = () => (
+  <>
+    <Meta
+      pageTitle="veOlas"
```
Comment (Tanya-atatakai) [inline apps/govern/pages/veolas.tsx:7]:
```suggestion
      pageTitle="veOLAS"
```

**Review by Tanya-atatakai — APPROVED** [review]
a small comment, approving in advance, otherwise looks very good! thanks

**Review by oaksprout — APPROVED** [review]

**Pull request merged**